### DISCUSSION TITLE
I wan to contribute

### DISCUSSION THREAD

**Discussion by ishanvaghani** [immich-app/immich#6088] [Q&A]
Hey There, Recently i started open source contribution. This project looks interesting to me. Not sure where to start can someone help here? If someone can give me small bug/feature then it will be helpful.

**Comment by bo0tzz** [comment]
The "good first issue" label is a nice place to start: https://github.com/immich-app/immich/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Comment by ishanvaghani** [comment]
Error (Xcode): ../../../../.pub-cache/hosted/pub.dev/photo_manager-2.8.1/lib/src/internal/image_provider.dart:71:5: Error: Type 'DecoderCallback' not found.

This error is coming while running project in ios simulator

**Comment by shenlong-tanwen** [comment]
> This error is coming while running project in ios simulator

The app is not yet compatible with the latest stable release of flutter. Can you check your flutter version with `flutter --version`? and if you're on 3.16, run `flutter downgrade` to go back to the previous stable release (3.13.x), after which, run `flutter clean`, `flutter pub cache clean` followed by `flutter pub get` and try rebuilding the app.

**Comment by ishanvaghani** [comment]
Thanks project is working; closing this
